src/index.js

```diff
@@ -1,56 +1,38 @@
 import './style.css';
 import Sync from './sync-solid.svg';
 import DotMenu from './ellipsis-v-solid.svg';
+import Trash from './trash-alt-regular.svg';
 import Enter from './left-arrow.png';
 
-class List {
-  constructor() {
-    this.listOfTasks = [];
-  }
-
-  addNewTask(newTask) {
-    return this.listOfTasks.push(newTask);
-  }
-}
-
-/* eslint max-classes-per-file: ["error", 2] */
-
 class Task {
   constructor(description, completed, index) {
     this.description = description;
     this.completed = completed;
     this.index = index;
   }
 }
+const input = document.querySelector('#enter-task');
+const listOfTasks = [];
 
-const list = new List();
+document.getElementById('sync-icon').src = Sync;
+document.getElementById('enter-icon').src = Enter;
 
-const input1 = 'Run 1 km';
-const input2 = 'Run 2 km';
-const input3 = 'Run 3 km';
+function addToLocalStorage(listOfTasks) {
+  localStorage.setItem('To-Do-List', JSON.stringify(listOfTasks));
+}
+
+const placeholder = document.querySelector('ul.placeholder');
 
-const createTask = (input) => {
-  const placeholder = document.querySelector('ul.placeholder');
-  const task = new Task(input, false, list.listOfTasks.length);
+function appendTaskToListAndUpdateLocalStorage() {
   const lastLi = document.createElement('li');
-  list.addNewTask(task);
-  placeholder.innerHTML = `
-    <li class="hello">
-      <p>Today's To-Do</p>
-      <img src="${Sync}" alt="Sync">
-    </li>
-    <li class="hello">
-      <input id="enter-task" placeholder="Add to your list...">
-      <img src="${Enter}" alt="Enter">
-    </li>
-  `;
-  list.listOfTasks.forEach((task) => {
+  localStorage.clear();
+  addToLocalStorage(listOfTasks);
+  listOfTasks.forEach((task) => {
     const li = document.createElement('li');
-    // Lodash, now imported by this script
     li.innerHTML = `
       <input type="checkbox" id="task" name="task-${task.index + 1}">
-      <label for="task-${task.index + 1}">${task.description}</label>
-      <img src="${DotMenu}" alt="Delete or Drag and drop">
+      <input type="text" value="${task.description}" id="${task.index}" class="task-text">
+      <img class="${task.index}" src="${DotMenu}" alt="Delete or Drag and drop">
     `;
     li.classList.add('hello');
```

Comment:
- [ ] [OPTIONAL] Please rename this class from 'hello' to something that has some semantic significance. the name of the class should usually describe or have a relationship to how it augments the element itself.